app/src/main/java/com/github/se/polyfit/data/local/database/MealDatabase.kt

```diff
@@ -75,15 +81,39 @@ class TimeConverter {
   }
 }
 
+class UriTypeConverter {
+  @TypeConverter
+  fun fromUri(uri: Uri): String {
+    return uri.toString()
+  }
+
+  @TypeConverter
+  fun toUri(uriString: String): Uri {
+    return Uri.parse(uriString)
+  }
+}
+
```

Comment:
Only strings are stored in the database. If the string `null` is stored in the database, it will be stored as `"null"`. The parse function will only throw an exception if the `Uri` is a null pointer. So, in theory, this should work properly.